### PR TITLE
Update protobuf types to match latest OpenMetrics RFC data model

### PR DESCRIPTION
**Pull request by robskillington**
cc @brian-brazil

### CODE REVIEW

File: proto/openmetrics_data_model.proto [hunk 1]
```diff
@@ -281,11 +278,12 @@ message SummaryValue {
   // This is for compatibility since some common systems do not expose them.
   uint64 count = 2;
 
-  // Optional.
+  // Optional, but SHOULD be specified.
```
Comment (brian-brazil) [inline proto/openmetrics_data_model.proto:281]: This is only a MAY, tending to SHOULD NOT. I'd not change this.

Comment (robskillington) [inline proto/openmetrics_data_model.proto:281]:
```suggestion
  // Optional.
```

File: proto/openmetrics_data_model.proto [hunk 2]
```diff
@@ -86,6 +68,21 @@ enum MetricType {
...
+  // Optional.
+  // Labels applying to all points, must be unique.
+  repeated Label labels = 1;
+
+  // Optional.
+  // If there is more than one point, all must have a timestamp field
+  // and they must be in increasing order of timestamp.
+  // If there is only one point and has no timestamp field, the receiver
+  // will assign a timestamp.
+  repeated Point point = 2;
```
Comment (robskillington) [inline proto/openmetrics_data_model.proto:83]:
```suggestion
  repeated Point points = 2;
```

File: proto/openmetrics_data_model.proto [hunk 3]
```diff
@@ -270,8 +267,8 @@ message InfoValue {
   repeated Label info = 1;
 }
 
-// Value for SUMMARY point. The start_timestamp only applies to the sum 
-// and count. The quantiles can be reset at arbitrary unknown times.
+// Value for SUMMARY point. The start_timestamp of the Point only applies to 
+// the sum  and count. The quantiles can be reset at arbitrary unknown times.
```
Comment (robskillington) [inline proto/openmetrics_data_model.proto:271]:
```suggestion
// the sum and count. The quantiles can be reset at arbitrary unknown times.
```

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**